frank-doc-doclet/src/main/java/org/frankframework/frankdoc/wrapper/FrankClassDoclet.java

```diff
@@ -341,7 +341,22 @@ private Boolean adapt(FrankClassDoclet c, Consumer<FrankClass> handler) {
 		handler.accept(c);
 		return null;
 	}
-	
+
+	@Override
+	public boolean extendsOrImplements(FrankClass ancestorCandidate) {
+		Function<FrankClassDoclet, Boolean> getter = c -> c.equals(ancestorCandidate) ? true : null;
```

Comment:
Je zou kunnen overwegen om in een volgend pr een 'opschoon' slag te maken door hier een `Optional<Boolean>` van te maken. Dan zou je code zoals
` if (optionalField.orElse(false)) { ` kunnen gebruiken in plaats van de if statements in de functies die aangeroepen worden.

Reply:
Ik heb dit punt ergens genoteerd. Je hebt aan de telefoon toegezegd dat ik het voor nu kan laten zitten.